src/server/kbn_server.js

```diff
@@ -141,7 +140,9 @@ export default class KbnServer {
 
     const { server, config } = this;
 
-    await new KibanaMigrator({ kbnServer: this }).migrateIndex();
+    if (server.kibanaMigrator) {
```

Comment:
When is server.kibanaMigrator null/undefined?

Reply:
If the elasticearch plugin is disabled. Honestly, I'm not sure what we ought to do in that scenario. It only comes up in testing (we have tests that launch Kibana w/out elasticsearch). An alternative approach would be to always define `kibanaMigrator` and just have it noop / warn if elasticsearch is disabled.

Reply:
Ah, that makes sense. In that case, I don't think my paranoid justifies a change here.